recipes/orange3-imageanalytics/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "Orange3-ImageAnalytics" %}
+{% set version = "0.1.13" %}
+{% set sha256 = "86a6bc65617b6796af764846d55e0c1bc49960716e673970a62beea3b6f60e10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
You probably also need `preserve_egg_dir: True` for the entry points to work correctly; I assume this is how Orange3 knows about the plugin.